### PR TITLE
Record testcase directory in judging runs.

### LINKED ISSUE #2059: Testcase in output/judgings/... directory is not easily matched with web UI
In the work directory of a judging, we have a directory per testcase which is of the form `testcase00189`. However, the number in there does not match the testcase rank in the problem and there's no (easy?) way to correlate this number to the testcase in the jury UI.

We should change this so that debugging is easier.

### PR DESCRIPTION
**Pull request by meisterT**
This makes it much easier to find out where to go for debugging.

Fixes #2059.

Example:
image

Clicking the copy/paste icon copies the full path.

### CODE REVIEW

**Comment by meisterT** [comment]
This is an alternative to https://github.com/DOMjudge/domjudge/pull/2211

**Review by nickygerritsen — APPROVED** [review]

**Review by vmcj — APPROVED** [review]
Fine by me, lets see which one @eldering wants.

File: webapp/src/Controller/Jury/SubmissionController.php [hunk 1]
```diff
@@ -394,6 +394,9 @@ public function viewAction(
                     '/\[output storage truncated after \d* B\]/',
                     (string)$runResult['output_run_last_bytes']
                 );
+                if ($firstJudgingRun) {
+                   $runResult['testcasedir'] = $firstJudgingRun->getTestcaseDir();
```
Comment (vmcj) [inline webapp/src/Controller/Jury/SubmissionController.php:398]:
```suggestion
                    $runResult['testcasedir'] = $firstJudgingRun->getTestcaseDir();
```

nitpick: phpcs wants 4 spaces here.

**Comment by eldering** [comment]
> Fine by me, lets see which one @eldering wants.

I think this solution is better than #2211 : both in the place/way the information is displayed and also by taking the actual directory from the judgehost instead of depending on configure/build time variables.

**Pull request merged**